MonoGame.Framework/Platform/Storage/StorageDevice.Native.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Xna.Framework.Storage
+{
+    partial class StorageDevice
```

Comment:
For the native platform you need to add a `Storage.Interop.cs` and forward all platform calls here to it.  See 
https://github.com/MonoGame/MonoGame/blob/develop/MonoGame.Framework/Platform/Native/Platform.Interop.cs

Reply:
I've added a placeholder `Storage.Interop.cs` file in the native sub-dir. But need to chat to you about what exactly I need to put in there. I'm still unclear how the Native stuff hangs together. Maybe if you have time tomorrow?

Reply:
As per Discord: With Native I've "left the methods there throwing NotImplemented for now" So may need more direction from you, as to what exactly needs to go in there, or if you have the cycles for you to implement that platform, perhaps.